electron_app/package.json

```diff
@@ -7,8 +7,13 @@
   "author": "Vector Creations Ltd.",
   "dependencies": {
     "auto-launch": "^5.0.1",
+    "electron": "^3.0.9",
```

Comment:
This would result in electron getting packaged within electron *unless* you have electron installed globally, as it'd get installed to node_modules which LITERALLY gets copied